projects/adt75/src/examples/dummy/dummy_example.c

```diff
@@ -0,0 +1,91 @@
+/***************************************************************************//**
```

Comment:
There was a discussion regarding the switch from renaming the example from "DUMMY" to "BASIC".